README.md

```diff
@@ -42,28 +42,12 @@ The `.zip` version will not automatically update.
 
 Using [Chocolatey](https://chocolatey.org)? Run `cinst Atom` to install the latest version of Atom.
 
-### Debian based (Debian, Ubuntu, Linux Mint)
+### Linux
 
-Atom is only available for 64-bit Linux systems.
-
-1. Download `atom-amd64.deb` from the [Atom releases page](https://github.com/atom/atom/releases/latest).
-2. Run `sudo dpkg --install atom-amd64.deb` on the downloaded package.
-3. Launch Atom using the installed `atom` command.
-
-The Linux version does not currently automatically update so you will need to
-repeat these steps to upgrade to future releases.
-
-### RPM based (Red Hat, openSUSE, Fedora, CentOS)
+You can learn how to configure your distribution's package manager to install and update Atom by reading the [Linux installation instructions](http://flight-manual.atom.io/getting-started/sections/installing-atom/#platform-linux) in the Flight Manual.  You will also find instructions on how to install Atom's official Linux packages without using a package repository.
 
 Atom is only available for 64-bit Linux systems.
 
-1. Download `atom.x86_64.rpm` from the [Atom releases page](https://github.com/atom/atom/releases/latest).
-2. Run `sudo rpm -i atom.x86_64.rpm` on the downloaded package.
-3. Launch Atom using the installed `atom` command.
-
-The Linux version does not currently automatically update so you will need to
-repeat these steps to upgrade to future releases.
-
 ### Archive extraction
 
 An archive is available for people who don't want to install `atom` as root.
```

Comment:
L63 should be clarified to state that the archive version doesn't automatically update.
(Technically, neither does the package manager version but it'd be nice to be more specific)